clang/lib/Sema/SemaCXXScopeSpec.cpp

```diff
@@ -767,6 +757,7 @@ bool Sema::BuildCXXNestedNameSpecifier(Scope *S, NestedNameSpecInfo &IdInfo,
   else
     Diag(IdInfo.IdentifierLoc, diag::err_undeclared_var_use)
         << IdInfo.Identifier;
+#endif
```

Comment:
You should probably remove these before merging :)

Reply:
Yeah, these changes shouldn't really be part of this PR. Just something I noticed when refactoring `BuildCXXNestedNameSpecifier` that needs to go. I'll address it in another PR.